palestirnative/palestirnative
# fix: separate serbia and montenegro

## Description

This PR fixes #26 

### How to test it

Open the suggest an alternative form, type Serbia it should appear alone, and type Montenegro it should be there too as a separate state.

### Screenshots
![image](https://github.com/palestirnative/palestirnative/assets/10698248/4fcf1bf7-0910-4621-ae4b-fdc35425b7ea)
